golang/cmd/kafka-to-postgresql/main.go

```diff
@@ -353,10 +316,10 @@ func PerformanceReport() {
 			"\n[HT] ProcessValueString queue lenght: %d"+
```

Comment:
Maybe use a different delimiter, `\n` will be shown and will not do a new line. Try something like |